sdk/cognitiveservices/azure-cognitiveservices-language-textanalytics/azure/cognitiveservices/language/textanalytics/_text_analytics_client.py

```diff
@@ -0,0 +1,328 @@
+                cls=language_result,
+                **kwargs
+            )
+        except ErrorException as error:
+            process_batch_error(error)
+
+    @distributed_trace
+    def recognize_entities(
+        self,
+        documents,  # type: Union[List[str], List[MultiLanguageInput], List[Dict[str, str]]]
+        model_version=None,  # type: Optional[str]
+        show_stats=False,  # type:  Optional[bool]
+        language=None,  # type: Optional[str]
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> List[Union[DocumentEntities, DocumentError]]
+        """Named Entity Recognition for a batch of documents.
+
+        The API returns a list of general named entities in a given document.
+        For the list of supported entity types, check:
+        https://aka.ms/taner
+        For the list of enabled languages, check:
+        https://aka.ms/talangs
+
+        :param documents: The set of documents to process as part of this batch.
+        :type documents:
+            list[str] or list[~azure.cognitiveservices.language.textanalytics.MultiLanguageInput]
+        :param str model_version: This value indicates which model will
+            be used for scoring. If a model-version is not specified, the API
+            will default to the latest, non-preview version.
+        :param bool show_stats: If set to true, response will contain document level statistics.
+        :param str language: The language for the entire batch. Per-document
+            language hints will take precedence over whole batch hints.
+        :return: The combined list of DocumentEntities and DocumentErrors in the order
+            the original documents were passed in.
+        :rtype: list[~azure.cognitiveservices.language.textanalytics.DocumentEntities,
+            ~azure.cognitiveservices.language.textanalytics.DocumentError]
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        docs = _validate_batch_input(documents, "language", language)
+        try:
+            return self._client.entities_recognition_general(
+                documents=docs,
+                model_version=model_version,
+                show_stats=show_stats,
+                cls=entities_result,
+                **kwargs
+            )
+        except ErrorException as error:
+            process_batch_error(error)
+
+    @distributed_trace
+    def recognize_pii_entities(
+        self,
+        documents,  # type: Union[List[str], List[MultiLanguageInput], List[Dict[str, str]]]
+        model_version=None,  # type: Optional[str]
+        show_stats=False,  # type:  Optional[bool]
+        language=None,  # type: Optional[str]
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> List[Union[DocumentEntities, DocumentError]]
+        """Recognize entities containing personal information for a batch of documents.
+
+        The API returns a list of personal information entities ("SSN",
```

Comment:
Here (and elsewhere) I think "The API returns..." is superfluous. Suggest "Return ..."